mlos_bench/mlos_bench/storage/util.py

```diff
@@ -30,7 +30,7 @@ def kv_df_to_dict(dataframe: pandas.DataFrame) -> Dict[str, Optional[TunableValu
         dataframe.rename(columns={'metric': 'parameter'}, inplace=True)
     assert dataframe.columns.tolist() == ['parameter', 'value']
     data = {}
-    for _, row in dataframe.iterrows():
+    for _, row in dataframe.astype('O').iterrows():
```

Comment:
Hmm, may need to add a test for this in mlos_bench as well.

Reply:
Looks like this util function is mostly used in displaying meta-data in the `mlos_bench/storage` layer. I can sift through and see if there is a reasonable location for a `dtype` test here.

Reply:
Eh, that's a little more complicated now that I look at it.  Let's take it as a separate item.  I think we can merge this one as is and circle back around to a proper test.  Thanks!

Reply:
https://github.com/microsoft/MLOS/issues/669